### DISCUSSION TITLE
KeyCloak and LDAP Connection

### DISCUSSION THREAD

**Discussion by pomland-94** [keycloak/keycloak#20198] [Q&A]
Hey,
is it possible that when I create Groups inside Keycloak that they are saved in my LDAP Structure, users are already saved there and this works great. But I wan't to have Keycloak as a Frontend for my LDAP user Managment, hope you know what I mean.

**Comment by slominskir** [comment]
See: https://github.com/keycloak/keycloak/issues/9609
